src/components/ProjectList/listOfProjects.js

```diff
@@ -482,6 +482,20 @@ const projectList = [
     githubLink: 'https://github.com/ifmeorg/ifme/',
     description: 'A community for mental health experiences',
     tags: ['ruby-on-rails', 'javascript', 'react']
-  }
+  },
+  {
```

Comment:
@aoberoi Instead of having two separate projects for Python and Node, can we directly call the project `SlackAPI`.
The image is the same, and the Github link for the project will be `https://github.com/slackapi`. 
The tags `python, http, javascript, typescript, api, sdk`.
WDYT?